nss-day-cohort-47/tabloidfullstack-bingo
# view user posts

# Description
User may now view their posts
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Login as a user => select "My Posts" from nav => only current logged in user's posts should render. 
# Checklist:
